packages/long-short-pair/src/mappings/longShortPairCreator.ts

```diff
@@ -0,0 +1,68 @@
+import { CreatedLongShortPair } from "../../generated/LongShortPairCreator/LongShortPairCreator";
+
+import { getOrCreateLongShortPairContract, getOrCreateToken } from "../utils/helpers";
+
+import { LongShortPair } from "../../generated/templates/LongShortPair/LongShortPair";
+
+import { log } from "@graphprotocol/graph-ts";
+
+// - event: CreatedLongShortPair(indexed address,indexed address,address,address)
+//   handler: handleCreatedLongShortPair
+
+export function handleCreatedLongShortPair(event: CreatedLongShortPair): void {
+  log.warning("lsp = {}, deployer = {}, longToken = {}, shortToken = {}", [
+    event.params.longShortPair.toHexString(),
+    event.params.deployerAddress.toHexString(),
+    event.params.longToken.toHexString(),
+    event.params.shortToken.toHexString(),
+  ]);
+  let contract = getOrCreateLongShortPairContract(event.params.longShortPair.toHexString());
+  let lspContract = LongShortPair.bind(event.params.longShortPair);
+
+  //TODO: Add the LSP creator (Factory) to this data type. I'm not exactly sure how to pull this info.
+  // contract.creator = event.address; something like this
```

Comment:
Yeah `event.address` should work, we use it [here](https://github.com/UMAprotocol/subgraphs/blob/master/packages/financial-contracts/src/mappings/perpetual.ts#L118)

Reply:
cool. will try again.